products_service/Dockerfile

```diff
@@ -11,22 +11,20 @@
 # CMD npm start
 
 FROM node:16.15.1-alpine
-COPY . /app
+
+RUN apk update && apk add bash postgresql-client
+RUN npm i nodemon -g
+
+USER node
 WORKDIR /app
 
-COPY package.json /app/package.json
-COPY package-lock.json /app/package-lock.json
+COPY --chown=node:node package.json .
+COPY --chown=node:node package-lock.json .
 
 # only needed when connecting to YB Managed nodes
 # COPY /certs /app/certs
 
 RUN npm install
```

Comment:
Should we use `npm ci` instead?

Reply:
Looks like that would be a good idea, especially if we're planning on moving to CI/CD.